README.md

```diff
@@ -38,7 +38,59 @@ View operator status:
 Inspect the status of the `machineconfigpool` objects which track upgrades:
 `oc describe machineconfigpool`
 
-(More to come)
+# Applying configuration changes to the cluster
+
+The MCO has "high level" knobs for some components of the cluster state;
+for example, SSH keys and kubelet configuration.  However, there are
+obviously a quite large number of things one may want to configure on
+a system.  For example, offline environments may want to specify
+an internal NTP pool.  Another example is static network configuration.
+By providing a MachineConfig object containing Ignition, arbitrary
+files can be laid down into `/etc` and `/var`.
+
+One known ergonomic issue right now is that you must encode file contents
+via [`data:` URIs](https://en.wikipedia.org/wiki/Data_URI_scheme). This is part
+of the specification of the [Ignition project](https://github.com/coreos/ignition).
+
+Note also the `mode` is in decimal; mode `420` is octal `0644` or `rw-r--r--`.
+
+This example MachineConfig object replaces `/etc/chrony.conf` with some
+custom NTP time servers; see
+[the chrony docs](https://chrony.tuxfamily.org/manual.html#Dial_002dup-configuration).
+
+```yaml
+# This example MachineConfig replaces /etc/chrony.conf
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 50-examplecorp-chrony
+spec:
+  config:
+    storage:
+      files:
+      - contents:
+          source: data:,server%20foo.example.net%20maxdelay%200.4%20offline%0Aserver%20bar.example.net%20maxdelay%200.4%20offline%0Aserver%20baz.example.net%20maxdelay%200.4%20offline
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/chrony.conf
+```
+
+The controller will notice the new MachineConfig and generate a new
+"rendered" version that looks like `worker-<hash>`.  Use
+`oc describe machineconfigpool/worker` to monitor the status of the rollout
+of the new rendered config to each node.
+
+Note this configuration only applies to workers; currently if
```

Comment:
Maybe put e.g. "(see the `role` label)" to be explicit about why it only applies to workers.